src/rlang/vec.h

```diff
@@ -266,6 +278,34 @@ r_obj* r_shared_lgl(bool x) {
   }
 }
 
+static inline
+bool _r_has_correct_length(r_obj* x, r_ssize n) {
+  return n < 0 || r_length(x) == n;
+}
+extern
```

Comment:
Why `extern`?

Reply:
oh it's not necessary here since it's not a variable. Not sure why I wrote this.